### ISSUE TITLE
BACKGROUND_TASKS.ASM missing

### ISSUE THREAD

**Issue by pdaehne** [zeropolis79/PETSCIIRobots-C64#2]
In line 4810, C64ROBOTS.ASM includes BACKGROUND_TASKS.ASM, but that file is missing in the repository.

**Comment by JesperJuul** [comment]
I've made it compile using the VIC-20 BACKGROUND_TASKS.ASM, but the program crashes to blue screen at startup. https://github.com/zeropolis79/PETSCIIRobots-Vic20/blob/main/BACKGROUND_TASKS.ASM

**Comment by pdaehne** [comment]
You're right, the C64 version uses exactly the same BACKGROUND_TASKS.ASM as the VIC-20 version.
I had to add a single line to C64ROBOTS.ASM (patch file is added below). After compiling the patched C64ROBOTS.ASM with ACME, I get a c64robots prg that is byte for byte identical to the version in the v1.2 disk image.

[C64ROBOTS.ASM.patch](https://github.com/zeropolis79/PETSCIIRobots-C64/files/13759533/C64ROBOTS.ASM.patch)

**Comment by JesperJuul** [comment]
What assembler are you using? On C64Studio 7.6, I finally made it work by adding a space before the quote in !PET and !SCR directives.
`TILENAME      !PET "tileset.gfx"`

**Comment by pdaehne** [comment]
ACME https://sourceforge.net/projects/acme-crossass/

**Comment by pdaehne** [comment]
In case someone is interested: Patch files to create source code for PETSCII Robots version 1.3.

[C64ROBOTS.ASM.v1.3.patch](https://github.com/user-attachments/files/16742197/C64ROBOTS.ASM.v1.3.patch)
[BACKGROUND_TASKS.ASM.v1.3.patch](https://github.com/user-attachments/files/16742201/BACKGROUND_TASKS.ASM.v1.3.patch)
